CMakeLists.txt

```diff
@@ -10,6 +10,14 @@ if(WIN32)
     endif()
 endif()
 
+if(APPLE)
+    if(NOT(ALGLIB_ROOT))
+        message(FATAL_ERROR "Didn’t work: passage of ALGLIB_ROOT to cmake. Download alglib and define its root directory in ALGLIB_ROOT")
```

Comment:
This is an ASCII file, but the addition of the U+2019 character changes that. It's the apostrophe in `Didn’t`. Rather change it to:
"Passage of ALGLIB_ROOT to CMake failed. Download ALGLIB and define its root directory in ALGLIB_ROOT"